buildSrc/src/main/java/org/elasticsearch/gradle/ElasticsearchDistributionType.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.gradle;
+
+public interface ElasticsearchDistributionType {
+    boolean shouldExtract();
```

Comment:
What do you think about defaulting this method and `isDocker()` to false?

Reply:
good idea. done